prow_config.yaml

```diff
@@ -26,6 +26,25 @@ workflows:
     params:
       installIstio: true
       workflowName: deployapp-istio
+  - app_dir: kubeflow/kubeflow/testing/workflows
+    component: kfctl_go_test
+    name: kfctl-go-iap-endpoint
+    job_types:
+      - postsubmit
```

Comment:
Lets also modify the existing kfctl_go_test to only run on presubmit. otherwise we are duplicating the test on postsubmit and perioidc runs.

Reply:
good point, fixed.